drivers/drivers_test.go

```diff
@@ -144,30 +147,45 @@ func TestMain(m *testing.M) {
 	}
 
 	for dbName, db := range dbs {
-		var ok bool
-		db.Resource, ok = pool.ContainerByName(db.RunOptions.Name)
-		if !ok {
-			buildOpts := &dt.BuildOptions{
-				ContextDir: "./testdata/docker",
-				BuildArgs:  db.BuildArgs,
+		dsn := db.DSN
+		var hostPort string
+		if db.RunOptions != nil {
```

Comment:
Maybe this whole block can be extracted to a method which would do an early return if `db.RunOptions` are missing?